content/en/blog/_posts/2023-06-29-container-image-signature-verification/index.md

```diff
@@ -0,0 +1,305 @@
+---
+layout: blog
+title: "Verifying container image signatures within CRI runtimes"
+date: 2023-06-29
+slug: container-image-signature-verification
+---
+
+**Author**: Sascha Grunert
+
+The Kubernetes community has been signing their container image-based artifacts
+since release v1.24. While the graduation of the [corresponding enhancement][kep]
+from `alpha` to `beta` in v1.26 introduced signatures for the binary artifacts,
+other projects followed the approach by providing image signatures for their
+releases, too. This means that they either create the signatures within their
+own CI/CD pipelines, for example by using GitHub actions, or rely on the
+Kubernetes [image promotion][promo] process to automatically sign the images by
+proposing pull requests to the [k/k8s.io][k8s.io] repository. A requirement for
+using this process is that the project is part of the `kubernetes` or
+`kubernetes-sigs` GitHub organization, so that they can utilize the community
+infrastructure for pushing images into staging buckets.
+
+[kep]: https://github.com/kubernetes/enhancements/issues/3031
+[promo]: https://github.com/kubernetes-sigs/promo-tools/blob/e2b96dd/docs/image-promotion.md
+[k8s.io]: https://github.com/kubernetes/k8s.io/tree/4b95cc2/k8s.gcr.io
```

Comment:
@saschagrunert , just want to verify if this is the correct link since it has a reference to k8s.gcr.io instead of https://github.com/kubernetes/registry.k8s.io

Reply:
Good point!

> NOTE: `k8s.gcr.io` is formerly the name of the Kubernetes upstream registry. For historical reasons, for some time this folder will remain named that.

Reply:
Yep, the reference is still valid :+1: